tests/lib/snaps/ovmf/snapcraft.yaml

```diff
@@ -0,0 +1,140 @@
+name: ovmf
```

Comment:
we can name it test-snapd-ovmf, update description to mention that that it's ovmf binaries with some testing keys enrolled used for snapd CI and push it to the store

Reply:
I've opened a name request for `test-snapd-ovmf`. Can move the snap files to `tests/lib/snaps/store/test-snapd-ovmf` ?